apps/rock-paper-scissors/src/utils/form.ts

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react'
+
+const isExists = (value: string) => (!value ? 'Field is required' : null);
+
+const isMinValue = (value: number, minDefaultValue: number) => (value < minDefaultValue ? `Minimal value ${minDefaultValue}` : null)
+
+const useForm = () => {
```

Comment:
It's a hook, why it's stored in util functions?
Also obviously not a form

Reply:
moved to hooks, change name